src/Microsoft.Azure.SignalR/EndpointUtility/ServiceEndpointUtility.cs

```diff
@@ -23,6 +27,12 @@ internal class ServiceEndpointUtility : IServiceEndpointUtility
 
         private static readonly string MissingRequiredProperty =
             $"Connection string missing required properties {EndpointProperty} and {AccessKeyProperty}.";
+        private static readonly string InvalidVersionValue =
+            $"Invalid value for {VersionProperty} property, value must follow regex: {ValidVersionRegex}.";
```

Comment:
We should simplify this error message? Simply tell user this version is not supported?